build.gradle

```diff
@@ -21,42 +21,19 @@ buildscript {
     maven { url "https://plugins.gradle.org/m2/" }
   }
   dependencies {
-    classpath "com.netflix.spinnaker.gradle:spinnaker-dev-plugin:5.2.2"
-    classpath "com.netflix.nebula:nebula-kotlin-plugin:$kotlinVersion"
-    classpath "org.jetbrains.kotlin:kotlin-allopen:$kotlinVersion"
+    classpath "com.netflix.spinnaker.gradle:spinnaker-dev-plugin:6.2.0"
```

Comment:
for 6.x (mostly because sorting out classpath conflicts from initscripts seems impossible) theres a bit extra here:
```
if (Boolean.valueOf(enablePublishing)) {
  classpath "com.netflix.spinnaker.gradle:spinnaker-gradle-project:6.2.0"
}
```

Reply:
also below after apply plugin: spinnaker.base-project
```
if (Boolean.valueOf(enablePublishing)) {
  apply plugin: 'spinnaker.project'
}
```

Reply:
add `enablePublishing=false` in `gradle.properties`

update the travis scripts to set `-PenablePublishing=true` instead of `-I gradle/init-publish.gradle`

Reply:
lastly remove any -Prelease.useLastTag options in gradle/*.sh except for the ones where we are executing candidate or final tasks